lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -230,9 +245,12 @@ def self.upload_binary(app_id, binary_path)
         end
       end
 
+      # Uploads the binary
+      #
+      # Returns the id of the release"> <explain "Only happens on a successful release
```

Comment:
You've still got some extra `"`s and `<`s

Can you explain what happens if the release is not successful?